docs-src/java/testing-activities.md

```diff
@@ -7,7 +7,7 @@ tags:
   - guide-context
 ---
 
-Mocking isolates code undergoing testing so the focus remains on the code, and not on external dependencies or other state. You can test activities using a mocked Activity environment. 
+Mocking isolates code undergoing testing so the focus remains on the code, and not on external dependencies or other state. You can test activities using a mocked Activity environment.
```

Comment:
Capital A in "You can test Activities"? To maintain consistency.

Reply:
Patrick is correct. This was not a change I made. I removed one sentence in one file (`docs-src/concepts/what-is-a-schedule-to-close-timeout.md`) so I assume everything else came along for the ride when I followed the steps in the README. I do agree that the capitalization contradicts our style, so I will fix that too.